release.config.cjs

```diff
@@ -0,0 +1,72 @@
+const transform = require('./release.utils.cjs');
```

Comment:
how can specify the initial version in semantic versions? I think we can start from 2.2.0